### PR TITLE
update instructions

### PR DESCRIPTION
**Pull request by riomyers**
changed to Chrome Web Store, added import instructions

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -133,11 +133,21 @@ To **generate a new Guld identity** and PGP key:
...
 1. Select the "import" radio button.
 
-1. ...
+1. Export the private key you used to register your Guld Identity from your preferred PGP key manager
+
+1. Copy and paste the exported private key into the text area labeled ASCII Armored Private Key
+
+1. Enter your existing Guld username
+
+1. Enter a strong passphrase for the private PGP key, and confirm it.
```
Comment (polarweasel) [inline docs/install.md]: "Enter the passphrase for the private PGP key." (There's no confirmation here, right?)

File: docs/install.md [hunk 2]
```diff
@@ -133,11 +133,21 @@ To **generate a new Guld identity** and PGP key:
...
+
+    Keep your passphrase stored securely! There is no way to recover a lost passphrase.
```
Comment (polarweasel) [inline docs/install.md]: Delete this note. They should know already they need to be careful with an existing key, right?

File: docs/install.md [hunk 3]
```diff
@@ -133,11 +133,21 @@ To **generate a new Guld identity** and PGP key:
...
+    
+1. Click the import button to import your private key and Guld Identity
+
+   **Note** If you get an error when trying to use your existing Guld username checkmark the Expert Mode (DANGER!) option and try the import again
```
Comment (polarweasel) [inline docs/install.md]: "If you get an error when trying to use your existing Guld username, check the "Expert Mode" box and try the import again." (fix the wording, and add period.)

**Review by polarweasel — CHANGES_REQUESTED** [review]

File: docs/install.md (hunk 1 above)
Comment (riomyers) [reply docs/install.md]: you're right! good catch

File: docs/install.md (hunk 2 above)
Comment (riomyers) [reply docs/install.md]: I would assume so but keeping this in helps us to not be held accountable by someone saying "well you didn't tell me it wasn't backed up", password recovery is a very common feature in centralized applications so someone may be under the impression it's stored somewhere

File: docs/install.md (hunk 3 above)
Comment (riomyers) [reply docs/install.md]: better phrasing, thank  you!  :blush:

File: docs/install.md (hunk 2 above)
Comment (polarweasel) [reply docs/install.md]: That's legit... leave it in!

Comment (riomyers) [reply docs/install.md]: good point!

**Pull request merged**